source/Nuke.Common.Tests/CI/ConfigurationGenerationTest.cs

```diff
@@ -216,6 +216,24 @@ yield return
             [Parameter("Azure Pipelines System Access Token")]
             public readonly string AzurePipelinesSystemAccessToken;
 
+            [Parameter("Test collections work as input")]
```

Comment:
Actually, `Parameter` only supports arrays.

Reply:
Removed values that aren't Array